cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/BrokerCapacity.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model.cruisecontrol;
+
+public class BrokerCapacity {
+    // CC allows specifying a generic "default" broker entry in the capacity configuration to apply to all brokers without a specific broker entry.
+    // CC designates the id of this default broker entry as "-1".
+    public static final int DEFAULT_BROKER_ID = -1;
+    public static final String DEFAULT_BROKER_DOC = "This is the default capacity. Capacity unit used for disk is in MiB, cpu is in percentage, network throughput is in KiB.";
+
+    public static final String DEFAULT_CPU_UTILIZATION_CAPACITY = "100";  // as a percentage (0-100)
+    public static final String DEFAULT_DISK_CAPACITY_IN_MIB = "100000";
+    public static final String DEFAULT_INBOUND_NETWORK_CAPACITY_IN_KIB_PER_SECOND = "10000";
+    public static final String DEFAULT_OUTBOUND_NETWORK_CAPACITY_IN_KIB_PER_SECOND = "10000";
+
+    private int id;
+    private String cpu;
+    private String disk;
+    private String inboundNetworkKiBPerSecond;
+    private String outboundNetworkKiBPerSecond;
```

Comment:
Maybe we want to align this field names to the corresponding one in the `BrokerCapacity` from the api?